Tests/Form/Type/AssertFormReferenceCompatibleWithSymfony28Trait.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Form\Type;
+
+trait AssertFormReferenceCompatibleWithSymfony28Trait
```

Comment:
Same as the code, we should not have special BC test cases or trait IMHO.

A test just has to check the targeted class. BC trick have to be done if needed.

The goal is just to test if your FormType works on each Symfony version, thanks to Travis.